src/Microsoft.Framework.Primitives/StringValues.cs

```diff
@@ -4,13 +4,14 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using Microsoft.Framework.Internal;
 
 namespace Microsoft.Framework.Primitives
 {
     /// <summary>
     /// Represents zero/null, one, or many strings in an efficient way.
     /// </summary>
-    public struct StringValues : IList<string>
+    public struct StringValues : IList<string>, IReadOnlyList<string>
```

Comment:
Could this just be an `IReadOnlyList<>`? We'd lose `ICollection.CopyTo` when we make the change - is there anything else that's interesting that might be affected with that change?